Provide a mechanism to allow updates in certain time window (#241)
bottlerocket-os/bottlerocket-update-operator: README.md

              value: "1"
```

##### Set Up Time Window
`PREVENT_UPDATE_SINCE` and `PREVENT_UPDATE_UNTIL` can be used to specify the prevent update time window during updating.
When `PREVENT_UPDATE_SINCE` and `PREVENT_UPDATE_UNTIL` is 0:0:0 (default), the feature is disabled.

> 

Reviewer: Nit: extra line here

To enable this feature, go to `bottlerocket-update-operator.yaml`, change `PREVENT_UPDATE_SINCE` and `PREVENT_UPDATE_UNTIL` to a `hour:minute:second` formate value (UTC (24-hour time notation)).

> Note: brupop uses UTC (24-hour time notation), please convert your local time to UTC.

Reviewer: One thing reading this doc that's not immediately clear to me is if this is _inclusive_ or _exclusive_ (although this is a bit of a minor concern).

If I set `PREVENT_UPDATE_SINCE` to `19:00` and `PREVENT_UPDATE_UNTIL` to `20:00`, what happens when the clock strikes `20:00` exactly?

it might be worth calling out if this is inclusive of the provided times or exclusive to eliminate any confusion.

              value: "1"
```

##### Set Up Update Time Window

> `UPDATE_WINDOW_START` and `UPDATE_WINDOW_STOP` can be used to specify the update time window during updating.

Reviewer 2: ```suggestion
`UPDATE_WINDOW_START` and `UPDATE_WINDOW_STOP` can be used to specify the time window in which updates are permitted.
```

When `UPDATE_WINDOW_START` and `UPDATE_WINDOW_STOP` is 0:0:0 (default), the feature is disabled.

To enable this feature, go to `bottlerocket-update-operator.yaml`, change `UPDATE_WINDOW_START` and `UPDATE_WINDOW__STOP` to a `hour:minute:second` formate value (UTC (24-hour time notation)). Note that `UPDATE_WINDOW_START` is inclusive and `UPDATE_WINDOW_STOP` is exclusive.

> For avoiding stopping in-process node update when update window stops, Bottlerocket update operator reserves 6 mins before `UPDATE_WINDOW_STOP` to finish remaining update circle.

Reviewer 2: ```suggestion
To avoid stopping an in-process node update when the update window stops, Bottlerocket update operator reserves 6 mins before `UPDATE_WINDOW_STOP` to finish remaining updates.
```
